content/posts/rh-distributed-tracing/index.mdx

```diff
@@ -0,0 +1,236 @@
+you might start [here](https://www.operate-first.cloud/developer). From there you can onboard your Kubernetes based cloud-native workloads with help
+from the Operate-First community.
+
+Recently, two operators were deployed on the Operate-First `Smaug` cluster that enable collection of OTLP data from
+any application. The operators are `Red Hat Distributed Tracing Platform` and `RedHat Distributed Tracing Data Collection`.
+The former includes a Jaeger operator for visualizing OTLP data and the latter includes the OpenTelemetry operator that
+manages namespaced OpenTelemetryCollectors to collect OTLP. These and many other Kubernetes operators are available from the
+[Certified OpenShift Operators Hub](https://catalog.redhat.com/software/operators/explore). This post describes how to connect your
+applications and workloads to these distributed tracing operators to reach your observability goals. You might bring your instrumented applications
+to Operate-First, or you might use this post to collect OTLP data from another K8s cluster.
+
+
+### Table of Contents
+* [Introduction](#introduction)
+* [Red Hat Distributed Tracing Data Collection](#red-hat-distributed-tracing-data-collection)
+* [Red Hat Distributed Tracing Platform](#red-hat-distributed-tracing-platform)
+* [Example Application](#example-application)
+* [Connecting to OpenTelemetry Collector](#connecting-to-opentelemetry-collector)
+* [Connecting to Jaeger](#connecting-to-jaeger)
+* [Visualizing Your Data](#visualizing-your-data)
+* [Summary](#summary)
+
+### Introduction
+
+Let’s start with what this blog is _not_. This post does not demonstrate how to instrument your applications to export OTLP.
+It assumes you have an application that is already instrumented. However, the [example application](https://github.com/sallyom/golang-ex.git)
+can be used as a guide for instrumenting a simple golang web server. There are also many online resources to help you along your way.
+Start with the [instrumentation docs](https://opentelemetry.io/docs/instrumentation/) for language-specific guides. With
+only a few lines of code, your application will be exporting telemetry. The interesting part is the collection and analysis of that
+data. It is a core mission of the OpenTelemetry project to make instrumenting easy and universal, to uncouple instrumentation from data storage
+and analysis.
+
+
+### Red Hat Distributed Tracing Data Collection
+
+The Red Hat Distributed Tracing Data Collection operator is an opinionated OpenTelemetry Collector Operator. It is easy to install from
+the OperatorHub as an OpenShift administrator. Once running, regular users can create its custom resources in their own projects. The operator
+watches all namespaces in the `Smaug` cluster for namespaced custom resources `OpenTelemetryCollector` and `Instrumentation`.
+In a cluster with this operator running, these resources can be found and created by navigating to the `API Explorer` in the cluster console.
```

Comment:
You can also access these from the left navigation menu `Installed Operators > <operat-name>` 
I think this is the primarily method end-users are expected to follow, and api explorer is generally the alternative

Reply:
updated! will need a new tag if it looks good, and thank you!